docs/io/ascii/write.rst

```diff
@@ -230,3 +230,208 @@ details.
+slightly from the CDS format in table description sections. CDS format includes more
+detailed description in the form of ``Abstract``, ``Notes``, ``References`` fields etc.
+and often has it in a separate file called ``ReadMe``. On the other hand, MRT format
+includes just the table ``Title``, ``Authors``, Table Caption and ``Notes`` and always
+has the ``ReadMe`` section together with the data.
+
+The :class:`~astropy.io.ascii.Cds` writer currently supports writing tables to MRT format.
+
+.. note::
+
+    The metadata of the table, apart from column ``unit``, ``name`` and ``description``,
+    will not be written in the output file. This also includes CDS/MRT format specific
+    ``ReadMe`` fields. The table lines where these fields can be filled in by
+    hand later, are however left empty.
+
+Examples
+""""""""
+
+..
+  EXAMPLE START
+  Writing CDS/MRT Format Tables Using astropy.io.ascii
+
+The command ``ascii.write(format='cds')`` writes an ``astropy`` `~astropy.table.Table`
+to the MRT format. Section dividers ``---`` and ``===`` are used to divide the table
+into different sections, with the last section always been the actual data.
+
+As the CDS/MRT standard requires,
+for columns that have a ``Unit`` attribute, the unit names are tabulated in the Byte-By-Byte
+description of the column. When columns do not contain any units, ``---`` is put instead.
+Also, a ``?`` is prefixed to the column description in the Byte-By-Byte for ``Masked``
+columns or columns that have null values, indicating them as such.
+
+The example below initializes a table with columns that have a ``Unit`` attribute and
+has masked values.
+
+  >>> from astropy.io import ascii
+  >>> from astropy.table import Table, Column, MaskedColumn
+  >>> from astropy import units as u
+  >>> table = Table()
+  >>> table['Name'] = ['ASASSN-15lh', 'ASASSN-14li']
+  >>> # MRT Standard requires all quantities in SI units.
+  >>> temperature = [0.0334, 0.297] * u.K
+  >>> table['Temperature'] = temperature.to(u.keV, equivalencies=u.temperature_energy())
+  >>> table['nH'] = Column([0.025, 0.0188], unit=u.Unit(10**22))
+  >>> table['Flux'] = ([2.044 * 10**-11] * u.erg * u.cm**-2).to(u.Jy * u.Unit(10**12))
+  >>> table['Flux'] = MaskedColumn(table['Flux'], mask=[True, False])
+  >>> table['magnitude'] = [u.Magnitude(25), u.Magnitude(-9)]
+
+Note that for columns with `~astropy.time.Time`, `~astropy.time.TimeDelta` and related values,
+the writer does not do any internal conversion or modification. These columns should be
+converted to regular columns with proper ``unit`` and ``name`` attribute before writing
+the table. Thus::
+
+  >>> from astropy.time import Time, TimeDelta
+  >>> from astropy.timeseries import TimeSeries
+  >>> ts = TimeSeries(time_start=Time('2019-1-1'), time_delta=2*u.day, n_samples=1)
+  >>> table['Obs'] = Column(ts.time.decimalyear, unit=u.day, description='Time of Observation')
```

Comment:
I see your point here and have made the correction.

> but again u.yr is 365.25 * 86400 sec, and 2019.0 is not 2019.0 * 365.25 * 86400 seconds since year 0.0.

However, doesn't the conversion of a `datetime` object to `decimalyear` uses this?